### ISSUE TITLE
[iOS] Event URL gets encoded

### ISSUE THREAD

**Issue by mathew3** [wmcmahan/react-native-calendar-events#272]
URL passed to `saveEvent` function gets encoded in iOS calendar.

URL passed
`https://example.com/events/44`

URL shown in the calendar event
`https%3A%2F%2Fexample.com%2Fevents%2F44`

### Environment

```
"react-native": "0.61.0"
"react-native-calendar-events": "^1.7.3"
```

### Steps to Reproduce

Pass URL within details to `saveEvent` function

### Expected Behavior

Used to work with RN < 0.60, clickable URL served as a deep link

### Actual Behavior

URL field in the calendar app

<img width="293" alt="Screenshot 2019-09-26 at 13 53 25" src="https://user-images.githubusercontent.com/15891489/65686515-0c0ce080-e065-11e9-846a-cd4eab9a05d9.png">

**Comment by jodelat** [comment]
problem still there for me...

**Comment by jodelat** [comment]
this fix still needs to be merged...

**Comment by jodelat** [comment]
@wmcmahan please merge in this fix

**Comment by MoOx** [comment]
Closed by 0045345743eee016afb98ffbce76dce1eb43fb17
